sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationAudience.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Data.AppConfiguration
+{
+    /// <summary> Cloud audiences available for AppConfiguration. </summary>
+    public readonly partial struct ConfigurationAudience : IEquatable<ConfigurationAudience>
```

Comment:
This will be an interesting one.  The client is `ConfigurationClient`, so the name choice here aligns with that.   The product is generally known as "AppConfiguration".   It's a 50/50 whether they'll ask to align to that or the client name.

I think what you've got is the right way to go and we should push to justify it based on alignment with the client name + options name.